pos_partner_location_map/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module allows to select partner address directly on map.
+For this Google Maps are used.
```

Comment:
> Since this module heavily depends on google, it should be renamed to something like `pos_partner_location_map_google`. Because there are other services like google maps. For example yandex maps. https://yandex.ru/dev/maps/

Hi @em230418 thank you for the feedback! This module implements core behaviour for both Google an OpenStreetMap. This is why we have opted for such a name keeping OpenStreetMap as a "Roadmap/Todo" item

Reply:
> Since this module heavily depends on google, it should be renamed to something like `pos_partner_location_map_google`. Because there are other services like google maps. For example yandex maps. https://yandex.ru/dev/maps/
> I think the name of the module should contain "google".

Hi @em230418 @legalsylvain  thank you for the feedback! This module implements core behaviour for any map provide. Google Is just the first one on the list. This is why we have opted for such a name keeping OpenStreetMap as a "Roadmap/Todo" item  So anyone could modify this module later without a need of creating a separate one for each map provider.

Reply:
> This module implements core behaviour for any map provide. Google Is just the first one on the list.

Do you plan to maintain behaviour of every map provider that is included to this module by others (1) or just google and OSM only (2)? If (2), you need to create seperate modules.

Reply:
> > This module implements core behaviour for any map provide. Google Is just the first one on the list.
> 
> Do you plan to maintain behaviour of every map provider that is included to this module by others (1) or just google and OSM only (2)? If (2), you need to create seperate modules.

Yes, this module is designed to be extendable with other map providers as well

Reply:
> Sorry, my question was incorrect.
> 
> > So anyone could modify this module later without a need of creating a separate one for each map provider.
> 
> Do you plan to maintain behaviour of every map provider that WILL BE included to this module (pos_partner_location_map) by others (1) or just google and OSM only (2)? If (2), you need to create separate modules.
> 
> > Yes, this module is designed to be extendable with other map providers as well
> 
> I don't have any doubt on this.

Yes if someone will open a PR with new map providers we will merge it when reviewed and approved.